scripts/check_commit_metadata.sh

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env bash
```

Comment:
I guess we have enough release related scripts to put them there, I'd still like to keep `/scripts/release.sh` (because used by humans) but the rest can go in `/scripts/release/*.sh`.